test/e2e/utils/kubernetes.go

```diff
@@ -486,6 +486,38 @@ func (k TestKubernetes) WaitForExternalService(ispn *ispnv1.Infinispan, timeout
 	return hostAndPort
 }
 
+// GetPVC returns a PVC bound with given PVC name
+func (k TestKubernetes) GetPVC(pvcName, namespace string) *corev1.PersistentVolumeClaim {
+	pvc := &corev1.PersistentVolumeClaim{}
+	err := k.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: pvcName}, pvc)
+	ExpectNoError(err)
+	return pvc
+}
+
+// GetSCList returns the default StorageClasses name in cluster.
+func (k TestKubernetes) GetSCList() string {
+	var defaultStorageClassName string
```

Comment:
Redundant var if we just return `""` at the end of the method.

Reply:
totally agree, thanks.